README.md

```diff
@@ -9,9 +9,6 @@ artifacts from OpenShift project.
  - Exports only `service`, `deploymentConfig`, `buildConfig`, `imageStream`, `replicationController`,
    `persistentVolumeClaim` for *Openshift*, other objects like `build`, `imageStreamTag`, `imageStreamImage`,
    `event`, `node`, `pod`, `persistentVolume`,  are not exported at the moment.
```

Comment:
I would mention here than some of them like 'pods' are not exported intentionally.

Also naming is inconsistent with Kubernetes:
_Services_ /  _service_
_ReplicationControllers_ / deploymentConfig  (first letter)